CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 0.0.8
+
+- ### Features
+
+  - **Support a variety of errors thrown from `getAssetFromKV` - [victoriabernard92], [issue/59] [pull/64]**
+
+    Support for throwing specific errors to allow the handler of `getAssetFromKV` to catch and differentiate them.
+    For example, a 404 `NotFoundError` error implies nothing went wrong, the asset just didn't exist while
+    a 500 `InternalError` means an expected variable was undefined.
```

Comment:
I think I'd recommend giving some context as to why this change was made and how asset handler worked before. 

"Previously, `getAssetFromKv` would throw the same error type if anything went wrong. Now it will throw different error types such as `NotFoundError` and `InternalError`." Maybe want to elaborate a bit more and maybe link to the place in the docs where we explain this.